keyboards/niu_mini/config.h

```diff
@@ -21,8 +21,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define VENDOR_ID       0x6E6D
```

Comment:
A few KBDFans boards already use `0x4B42  // kb`.
If this board is exclusive to them, I think that would be a more appropriate VID, otherwise I'm fine with the proposed one.

Reply:
That was my original plan, but since the niu mini isn't under the KBDFans folder in QMK, I felt it more appropriate to give it its own VID.